community/compton/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+export NO_DBUS=1
+
+# fake asciidoc
+touch a2x asciidoc man/compton.1 man/compton-trans.1 ; chmod +x a2x asciidoc
+PATH=$PWD:$PATH
+
+make DESTDIR="$1" install
```

Comment:
Missing standalone `make`.

Reply:
Standalone `make` should also allow you to remove the fake `asciidoc` portions.

Reply:
unfortunately not the case for me

ompton.o opengl.o c2.o -lGL -lm -lrt -lpcre -lXcomposite -lXdamage -lXfixes -lXrender -lX11 -lXext -lXrandr -lXinerama -lconfig 
a2x --format manpage man/compton.1.asciidoc
make: a2x: Command not found
make: *** [Makefile:121: man/compton.1] Error 127
!> compton Build failed

Reply:
Maybe a simple `sed` could suffice to change `a2x` to `:` (noop).

Reply:
fixed